src/server/actions/EventVolunteers.ts

```diff
@@ -19,3 +19,18 @@ export async function getEventVolunteersByRole(
   }
   return evs;
 }
+
+export async function getEventVolunteersByOrganization(
+  organizationId: string
+): Promise<EventVolunteerResponse[]> {
+  let evs: EventVolunteerResponse[];
+  try {
+    await dbConnect();
+    evs = await EventVolunteerSchema.find({
+      organization: organizationId,
+    }).populate('volunteer');
+  } catch (error) {
+    throw new CMError(CMErrorType.InternalError);
+  }
+  return evs;
+}
```

Comment:
i dont think you needed to write this. could you not use `getVolunteersByOrganization` from `src/server/actions/Volunteer.ts` (this might have been a discoverability problem. we may need to move this function to make it more obvious that it exists)

Reply:
No. `getVolunteersByOrganization` gets volunteers, not event volunteers.